tests/rt.conf

```diff
@@ -33,6 +33,7 @@ RUN     | cpld_debug_p8
 COMPILE | -DAPP=ATM -DCCPP_SUITES=FV3_GFS_v16,FV3_GFS_v15_thompson_mynn,FV3_GFS_v15_thompson_mynn_RRTMGP,FV3_GFS_v16_p7_rrtmgp,FV3_GFS_v16_p8 -D32BIT=ON   |                | fv3 |
 RUN     | control                                                                                                                 |                                         | fv3 |
 RUN     | control_decomp                                                                                                          |                                         |     |
+RUN     | control_2ddecomp                                                                                                        |                                         |     |
```

Comment:
@junwang-noaa Would we expect  post results from control and control_2ddecomp are identical? Same expectation for regional_control and regional_control_2ddecomp?

Reply:
I would rename this test to `control_2dpostdecomp`, to indicate that '2d' is post's or (write) decomposition, not model's decomposition (i.e. layout)

Reply:
@WenMeng-NOAA Yes,  control and control_2ddecomp generate same results. Please see the "CNTL_DIR" in the test definition below. The control_2ddecomp is comparing results with control test(CNTL_DIR=control). The same for the regional 2ddecomp test.
@DusanJovic-NOAA renamed.